velox/functions/lib/aggregates/MinMaxByAggregatesBase.h

```diff
@@ -324,9 +327,12 @@ class MinMaxByAggregateBase : public exec::Aggregate {
         decodedComparison_.isNullAt(0)) {
       return;
     }
+
+    const auto* indices = decodedComparison_.indices();
     if (decodedValue_.mayHaveNulls() || decodedComparison_.mayHaveNulls()) {
       rows.applyToSelected([&](vector_size_t i) {
-        if (decodedComparison_.isNullAt(i)) {
+        if (velox::aggregate::prestosql::checkNestedNulls(
```

Comment:
I just noticed that this code is in lib, not prestosql. Hence, it should not have Presto-specific logic and cannot depend on code from prestosql.

One option might be to move checkNestedNulls to lib.

Reply:
Done, moved to lib.